client/src/App/index.js

```diff
@@ -7,12 +7,14 @@ import FavoritePage from '../container/FavoritePage';
 import DetailsPage from '../container/DetailsPage';
 import { Login } from '../components';
 import { AuthProvider } from '../container/authContext';
+import Header from '../components/header';
```

Comment:
Component names should follow the convention. (Header instead of header).
```suggestion
import Header from '../components/Header';
```